argus/scenarios/cloud/windows.py

```diff
@@ -14,9 +14,30 @@
 #    under the License.
 
 import collections
+from six.moves.urllib.parse import urlparse

```

Comment:
It actually doesn't, already ran the coding style linters over the patch to make sure.